### DISCUSSION TITLE
Fulfilling the criteria for chrome autoplay

### DISCUSSION THREAD

**Discussion by redstonerti** [bevyengine/bevy#13524] [Q&A]
I have been unable to get sound to work on my browser because the user is supposed to interact with the website first.

I assume I would have to communicate with the website to get this to work, but I have no idea how to do that.

I would also have to reinitialize the AudioContext, which I also don't know how to do.

Does anyone have any ideas?

**Comment by mockersf** [comment]
the easiest workaround is to use the javascript from https://developer.chrome.com/blog/web-audio-autoplay/#moving-forward
